rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -308,6 +303,270 @@ namespace rocal{
             .def_readwrite("decode_time",&TimingInfo::decode_time)
             .def_readwrite("process_time",&TimingInfo::process_time)
             .def_readwrite("transfer_time",&TimingInfo::transfer_time);
+        py::class_<rocalTensor>(m, "rocalTensor")
+            .def(
+                "max_shape",
+                [](rocalTensor &output_tensor) {
+                    return std::make_pair(output_tensor.info().max_shape().at(0), output_tensor.info().max_shape().at(1));
+                },
+                R"code(
+                Returns a tensor buffer's width.
+                )code"
+            )
+            .def(
+                "batch_size",
+                [](rocalTensor &output_tensor) {
+                    return output_tensor.info().dims().at(0);
+                },
+                R"code(
+                Returns a tensor batch size.
+                )code"
+            )
+            .def(
+                "color_format",
+                [](rocalTensor &output_tensor) {
+                    switch(output_tensor.info().color_format())
```

Comment:
Is the color_format used in python?

Reply:
It was used earlier. Now since we changed to get the dimensions[] directly. We dont use this API explicitly. Shall remove this API